From 54943f866a295256e7c88714e9dd4e6340cbc2c2 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Tue, 30 Oct 2018 08:56:14 +0200
Subject: [PATCH 0741/1051] arm64: dts: marvell: armada-3720-B: disable USB3
 controller

On setup B, USB3 controller is not available. Disable it.

Change-Id: I0d71592d8528e9ea23b2af3dc07d16de4829e07c
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60602
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db-B.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
index 6b2e9d53ddca..427724d9d21f 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
@@ -23,3 +23,7 @@
 		reg = <1>;
 	};
 };
+
+&usb3 {
+	status = "disabled";
+};
-- 
2.17.1

